docs/source/spec/aws-core.rst

```diff
@@ -716,6 +716,102 @@ applied through the ``aws.api#data`` trait.
         permissions.
 
 
+.. _aws.api#controlPlane-trait:
+
+------------------------------
+``aws.api#controlPlane`` trait
+------------------------------
+
+Summary
+    Indicates that a service, resource, or operation is considered part of
+    the *control plane*.
+Trait selector
+    ``:test(service, resource, operation)``
+Value type
+    Annotation trait
+Conflicts with
+    :ref:`aws.api#dataPlane-trait`
+
+This trait is effectively inherited by shapes bound within a service or
+resource. When applied to a service or resource shape, all resources and
```

Comment:
Shouldn't the data plane and `controlPlane` pieces in this sentence be swapped? Everything under is "CONTROL" unless marked "DATA"?